tailwind.config.js

```diff
@@ -58,6 +58,12 @@ module.exports = {
       primary: colors.indigo,
       success: colors.green,
       error: colors.red,
+      blue: colors.blue,
```

Comment:
I don't follow here. `Blue` is shorthand for color blue already, is this needed? And then you are extending the theme so that blue is actually indigo.